src/tiled/tileanimationeditor.ui

```diff
@@ -20,6 +20,35 @@
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
+         <widget class="QSpinBox" name="frameTime">
+             <property name="suffix">
+              <string>ms</string>
+             </property>
+             <property name="minimum">
+              <number>20</number>
+             </property>
+             <property name="maximum">
+              <number>10000</number>
+             </property>
+             <property name="singleStep">
+              <number>20</number>
+             </property>
+             <property name="value">
+              <number>100</number>
+             </property>
+         </widget>
```

Comment:
You would generally use Qt Designer or the Design mode in Qt Creator for editing `.ui` files, but if you do it by hand, please do take care that they are formatted consistently (easiest should be to open the file in Qt Designer and save it again).